yaksh/templates/yaksh/course_detail.html

```diff
@@ -7,50 +7,80 @@
 {% block css %}
 <link rel="stylesheet" media="all" type="text/css" href="{{ URL_ROOT }}/static/yaksh/css/course.css" />
 {% endblock %}
-
+{% block script %}
+<script language="JavaScript" type="text/javascript" src="{{ URL_ROOT }}/static/yaksh/js/jquery-1.4.2.min.js"></script>
+<script language="JavaScript" type="text/javascript" src="{{ URL_ROOT }}/static/yaksh/js/course.js"></script>
+{% endblock %}
 {% block manage %}
 <div class="row">
     <div class="span6 wrap">
         <center><b><u>Requests</u></b></center><br>
+        {% if course.get_requests %}
+        <input type="checkbox" class="checkall"/>&nbsp;<font size="2">Select all</font>
+        <div id="enroll-all">
+        <form action="{{URL_ROOT}}/exam/manage/enroll/{{ course.id }}/" method="post">
+        {% csrf_token %}
         {% for request in course.get_requests %}
         <div class="well">
             <div class="row">
                 <div class="span3" style="padding-top:10px">
-                    {{ request.username }}
+                    <input type="checkbox" name="check" value="{{ request.id }}">&nbsp;<a href="#" data-toggle="tooltip" title=" Full Name: {{ request.get_full_name }}&#013;Institute: {{ request.profile.institute }}&#013;Department: {{ request.profile.department}}&#013;Roll Number: {{ request.profile.roll_number }}">&nbsp;{{ request.username }}</a>
```

Comment:
You can give tooltip to all the users that we see in the course, whether in request or rejected or enrolled.